frontend/packages/ceph-storage-plugin/integration-tests/tests/ocp-tests/installFlow.scenario.ts

```diff
@@ -0,0 +1,96 @@
+  ocsOperator,
+  createLink,
+  sizeDropdown,
+  optionSmallSize,
+  searchInputOperators,
+  primaryButton,
+} from '../../views/installFlow.view';
+import { goToInstalledOperators, getStorageClusterLink } from '../../views/add-capacity.view';
+import { waitFor, hasTaints, hasOCSTaint, refreshIfNotVisible } from '../../utils/helpers';
+
+const Installer = new InstallCluster(NS);
+
+describe('Testing OCS Cluster Creation', () => {
+  beforeAll(async () => {
+    await Installer.createNamespace();
+    expect(browser.getCurrentUrl()).toContain(NS);
+    await Installer.subscribeToOperator(CATALOG_SRC);
+  }, 2 * MINUTE);
+
+  it(
+    'Test Installation for OCS operator',
+    async () => {
+      await goToInstalledOperators();
+      await searchInputOperators.clear();
+      await searchInputOperators.sendKeys(OCS_OPERATOR_NAME);
+      await waitFor(ocsOperatorStatus, SUCCESS, 2);
+      await click(ocsOperator);
+      await browser.wait(until.and(crudView.untilNoLoadersPresent));
+      // This can fail in a fresh cluster
+      try {
+        await browser.wait(until.visibilityOf(createLink), 10 * SECOND);
+      } catch {
+        await refreshIfNotVisible(createLink, 5);
+      }
+      const storageClusterLink = await getStorageClusterLink();
+      await click(storageClusterLink);
+      await browser.wait(until.and(crudView.untilNoLoadersPresent));
+      // Node list fluctuates
+      await browser.sleep(5 * SECOND);
+      const nodes = await selectWorkerRows();
+      await click(sizeDropdown);
+      await click(optionSmallSize);
+      await browser.wait(until.elementToBeClickable(primaryButton));
+      await click(primaryButton);
+      // Letting it move to another page before waiting for loaer
+      await browser.sleep(SECOND);
+      await browser.wait(until.and(crudView.untilNoLoadersPresent));
+
+      // Required for the storageCluster YAML to populate only required in a fresh subscription (Fix this in the install page)
+      await browser.sleep(10 * SECOND);
+      // defualt sc
+      const defaultSC = execSync(`oc get storageclasses | gawk '{print $1 $2}' | grep default`);
+      const storageCR = JSON.parse(
+        execSync(`kubectl get storageclusters ${STORAGE_CLUSTER_NAME} -n ${NS} -o json`).toString(),
+      );
+      // Tests for storage class request is correct
+      expect(
+        defaultSC.includes(
+          storageCR?.spec?.storageDeviceSets?.[0]?.dataPVCTemplate?.spec?.storageClassName,
+        ),
+      ).toBe(true);
+      // Tests for storage class
+      expect(
+        storageCR?.spec?.storageDeviceSets?.[0]?.dataPVCTemplate?.spec?.resources?.requests
+          ?.storage,
+      ).toEqual('512Gi');
+      nodes.forEach((node) => {
+        const nodeJSON = JSON.parse(execSync(`oc get nodes ${node} -o json`).toString());
+        const labelKeys = Object.keys(nodeJSON.metadata.labels);
+        expect(labelKeys).toContain(OCS_NODE_LABEL);
+        expect(labelKeys).toContain(WORKER_LABEL);
```

Comment:
Why are checking this label?

Reply:
Its need not to be worker node. Even master can be `schedule-able`. Just checking the taints is fine.

Reply:
ack